package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "leaderboard-api",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "private": true,
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "webpack"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/akhror-valiev/leaderboard-api.git"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/akhror-valiev/leaderboard-api/issues"
+    },
+    "homepage": "https://github.com/akhror-valiev/leaderboard-api#readme",
+    "devDependencies": {
+        "babel-eslint": "^10.1.0",
+        "eslint": "^7.32.0",
+        "eslint-config-airbnb-base": "^14.2.1",
+        "eslint-plugin-import": "^2.26.0",
+        "hint": "^7.1.3",
+        "stylelint": "^13.13.1",
+        "stylelint-config-standard": "^21.0.0",
+        "stylelint-csstree-validator": "^1.9.0",
+        "stylelint-scss": "^3.21.0"
+    },
+    "dependencies": {
+        "css-loader": "^6.7.2",
+        "html-webpack-plugin": "^5.5.0",
+        "lodash": "^4.17.21",
+        "style-loader": "^3.3.1",
+        "webpack": "^5.75.0",
+        "webpack-cli": "^5.0.0"
```

Comment:
- [ ] Good job setting up the webpack, however please make sure All webpack related packages are included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, these packages should be added as devDependencies since they are only required in development environment, right now they are added as dependencies it is an easy fix just run `npm uninstall webpack` `npm uninstall webpack-cli` and then run `npm install webpack --save-dev` `npm install webpack-cli --save-dev` and  `npm install webpack-dev-server --save-dev` this will  ad these packages in devDependencies.